anyway/infographics_utils.py

```diff
@@ -1118,22 +1120,48 @@ def localize_items(request_params: RequestParams, items: Dict) -> Dict:
 
 
 @register
-class InjuredAccidentsWithPedestriansWidget(SubUrbanWidget):
+class InjuredAccidentsWithPedestriansWidget(UrbanWidget):
     name: str = "injured_accidents_with_pedestrians"
 
+    def validate_parameters(self):
+        # TODO: validate each parameter and display message accordingly
+        return self.request_params.location_info.get('yishuv_name') is not None and \
+            self.request_params.news_flash_obj.street1_hebrew is not None and  \
+            self.request_params.years_ago is not None
+
     def __init__(self, request_params: RequestParams):
         super().__init__(request_params, type(self).name)
+
         self.rank = 18
-        self.text = {"title": "נפגעים הולכי רגל ברחוב ז׳בוטינסקי, פתח תקווה"}
+        self.text = {"title": f"נפגעים הולכי רגל ב- {get_news_flash_location_text(request_params.news_flash_obj)}"}
 
     @staticmethod
+    # TODO: change?
     def is_in_cache() -> bool:
         return False
 
     def generate_items(self) -> None:
-        self.items = (
-            InjuredAccidentsWithPedestriansWidget.injured_accidents_with_pedestrians_mock_data()
-        )
+        try:
+            if not self.validate_parameters():
+                logging.exception(f"Could not validate parameters for {NewsFlash} : {self.request_params.news_flash_obj.id}")
+                return None
+
+            query = db.session.query(InvolvedMarkerView)    \
+                .with_entities(InvolvedMarkerView.accident_year,    \
+                    InvolvedMarkerView.injury_severity, \
+                    InvolvedMarkerView.injury_severity_hebrew,\
+                    func.count(distinct(f"{InvolvedMarkerView.provider_and_id}_{InvolvedMarkerView.involve_id}")))  \
```

Comment:
@atalyaalon @ziv17 distinct for strings doesn't work, I wanted to ask if the purpose of the following line fro the redash query is to perform distinct on BOTH provider_and_id and involve_id
![image](https://user-images.githubusercontent.com/24354642/133778881-fc56323c-d10d-4a67-8b2e-5d35651b46e5.png)

Reply:
After `injury_severity_hebrew` is removed, there will be no strings.

Reply:
Removed distinct altogether following our talk @atalyaalon